README.md

```diff
@@ -1,14 +1,11 @@
 # travis-pyenv
-Set up [pyenv](https://github.com/yyuu/pyenv) to use in [Travis CI](https://travis-ci.org) builds.
-
-Setting up pyenv properly in a Travis CI build environment can be quite tricky. This repo contains a script ([`setup-pyenv.sh`](setup-pyenv.sh)) that makes this process much simpler.
-
-A common use case for this is to install an up-to-date version of [PyPy](http://pypy.org). The Travis CI build images currently contain a very old version of PyPy which breaks some common Python modules.
+With [Travis CI](https://travis-ci.org), sometimes you want to test your project code using an unsupported Python build, such as an exact version of [CPython](http://www.python.org), a more recent version of [PyPy](http://pypy.org), or any number of other Python implementations. You can use [pyenv](https://github.com/yyuu/pyenv) to install a very specific Python version or distribution, but setting it up properly in a Travis CI build environment can be tricky. This repo contains a script ([`setup-pyenv.sh`](setup-pyenv.sh)) you can download and call in your `.travis.yml` configuration to simplify this process.
 
 ## Usage
-1. Set the `$PYENV_VERSION` environment variable to the Python to install.
-2. Tell Travis to cache the `$HOME/.pyenv_cache` directory.
+1. Instead of setting a value for `python` in your `.travis.yml` build matrix, use the `$PYENV_VERSION` environment variable to set the pyenv Python version you want to use for a given build.
```

Comment:
I'm not sure about prescribing this way of using the script. This suggests that the user would use `travis-pyenv` for all the build environments of the project.

While that may sometimes be the case, I think the more common use case is that someone needs to install one or two obscure Python implementations (e.g. PyPy) but is otherwise fine with using Travis' available Python versions. Hence why the example Travis does a `if [[ -n "$PYENV_VERSION" ]]; then` instead of just using the script by default.